core/src/prediction/LocalLinearPredictionStrategy.cpp

```diff
@@ -76,24 +76,27 @@ std::vector<double> LocalLinearPredictionStrategy::predict(
   }
 
   // find ridge regression predictions
-  Eigen::MatrixXd M (num_variables+1, num_variables+1);
-  M.noalias() = X.transpose()*weights_vec.asDiagonal()*X;
+  Eigen::MatrixXd M_penalized(num_variables+1, num_variables+1);
+  M_penalized.noalias() = X.transpose()*weights_vec.asDiagonal()*X;
```

Comment:
Should this be M_unregularized rather than penalized?

Reply:
Yes, thanks! Was rushing on this a bit :)